### PR TITLE
Immersive navbar top padding

### PR DESCRIPTION
**Pull request by mbardakovD2L**
Immersive navbar was overlapping the title of the fullscreen view instead of pushing the content down, so I added padding to compensate. I'm sure there's a cleaner way to do this, but I couldn't think of one and wanted to get this out ASAP.

See rally story: https://rally1.rallydev.com/#/357251704080ud/custom/367300408400?detail=%2Fuserstory%2F460369362992

### CODE REVIEW

File: components/d2l-work-to-do/d2l-work-to-do.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ class WorkToDoWidget extends EntityMixinLit(LocalizeWorkToDoMixin(LitElement)) {
 				}
 				.d2l-work-to-do-fullscreen-container {
 					background-image: linear-gradient(to bottom, #f9fbff, #ffffff);
-					padding: 0 2rem;
+					padding: 67px 2rem 0;
```
Comment (JasperAtD2L) [inline components/d2l-work-to-do/d2l-work-to-do.js:97]: How'd you come up with 67?

Comment (mbardakovD2L) [reply components/d2l-work-to-do/d2l-work-to-do.js:97]: Inspected the height of the navbar itself in chrome

Comment (surags) [reply components/d2l-work-to-do/d2l-work-to-do.js:97]: Would that vary based on the window size?

Comment (JasperAtD2L) [reply components/d2l-work-to-do/d2l-work-to-do.js:97]: No, it'll vary based on the DPI of the person's display. It should probably be a rem.

Comment (mbardakovD2L) [reply components/d2l-work-to-do/d2l-work-to-do.js:97]: rem makes more sense - I'll figure out what the conversion is and then update that

File: components/d2l-work-to-do/d2l-work-to-do.js [hunk 2]
```diff
@@ -94,7 +94,7 @@ class WorkToDoWidget extends EntityMixinLit(LocalizeWorkToDoMixin(LitElement)) {
 				}
 				.d2l-work-to-do-fullscreen-container {
 					background-image: linear-gradient(to bottom, #f9fbff, #ffffff);
-					padding: 67px 2rem 0;
+					padding: 3.35rem 2rem 0;
```
Comment (JasperAtD2L) [inline components/d2l-work-to-do/d2l-work-to-do.js:97]: nit: not sure how it's done elsewhere, but expanding to the full 4 piece padding, IMHO is easier to look at than a 3 piece padding. With 2 it's easy to see top/bottom vs left/right.

Comment (mbardakovD2L) [reply components/d2l-work-to-do/d2l-work-to-do.js:97]: that's totally fair - I have to google it every time so I'm all in favour of whatever's easier

**Review by JasperAtD2L — APPROVED** [review]

**Pull request merged**